content/documentation/multiformat-output.md

```diff
@@ -1,12 +1,218 @@
+Create the HTML files for your project by running `quire site` in your command-line shell. The files will be built into your project’s `site` folder along with all the necessary static assets like image files, stylesheets and script files. The `site` file will be updated and overwritten each time you run `quire site`.
+
+{{< q-class "box tip" >}}
+- If you are including PDF and e-book downloads as part of your online site (this is Quire’s default) you’ll need to update those files by running `quire epub`, `quire mobi`, and `quire pdf` as described below **before** running `quire site`.
+{{< /q-class >}}
+
+You can hide specific pages of your project from the site output by adding `online: false` to the page YAML. Or conversely, you can add pages exclusively to the site output, by adding `epub: false` and `pdf: false` to the page YAML and leaving the `online` attribute unset, or set to `true`.
+
+Unlike `epub: false` and `pdf: false`, adding `online: false` does not stop a page from being built and included in the site output. Rather, it just unlinks those pages and adds a metadata tag to them to prevent them from being indexed by search engines. You may want to manually delete the pages from the `site` folder after outputting and before uploading to a web server if you want to be fully certain they can’t be accessed online under any circumstances.
+
+{{< q-class "box tip" >}}
+- Links to download the EPUB, MOBI, and PDF versions are automatically included in the sidebar menu of the online version of your project. These links can be removed or modified by making changes to the `resource_link` information in your `data/publication.yml` file.
+- You can add links to these files from anywhere in your markdown files by linking to `/downloads/output.epub`, `/downloads/output.mobi`, and `/downloads/output.pdf` respectively.
+{{< /q-class >}}
+
+## Deploy Your Project on Online
+
+A Quire site is designed to be hosted on virtually any web server, either one your institution already runs, or a new server from any hosting service you might sign up for. You do not need any special back-end setup.
+
+When hosting a site this way, you will typically follow these steps:
+
+1. Output your PDF and e-book files following the directions above, if you are going to include them as part of your published project
+2. Update the baseURL in config/site.yml to match the URL where the site will ultimately be hosted
+3. Output your site following the directions above
+4. Upload the contents of the site folder to your web host based on the directions they provide
+
+If you do not already have a web server or hosting plan, we’ve found that [Netlify](https://www.netlify.com/?_ga=2.127713933.1882208577.1616011630-1141534382.1615320952) offers a couple excellent options.
```

Comment:
The Netlify link came with a lot of crud, probably from copy-paste, that should be deleted: "/?_ga=2.127713933.1882208577.1616011630-1141534382.1615320952"